src/frontends/pytorch/src/op/slice_scatter.cpp

```diff
@@ -26,7 +26,7 @@ OutputVector translate_slice_scatter_fx(const NodeContext& context) {
     ov::Output<ov::Node> dim;
     if (!context.input_is_none(2)) {
         dim = context.get_input(2);
-        if (dim.get_partial_shape().rank().is_dynamic() || dim.get_partial_shape().rank().get_length() == 0) {
+        if (!(dim.get_partial_shape().rank().is_dynamic()) && dim.get_partial_shape().rank().get_length() == 0) {
```

Comment:
This change is dangerous. This is a common code and by default we do Unsqueeze for dynamic type